From: John L. Jolly <jjolly@suse.com>
Date: Fri Oct 5 22:31:42 2018 +0000
Subject: Remove shebang from scripts only intended to be sourced
Patch-mainline: Upstream
Git-repo: https://github.com/warewulf/warewulf3
Git-commit: 9edf6569be90949af3acc06cfe629a319dcfe364
References: 

When a shell script is not intended to be executed as a stand-alone
executable, the shebang at the start of the script is not necessary. In
cases where the script should not be executed, the shebang should be
removed. The following scripts are specifically sourced by other
executable scripts and thus have no need for the shebang:

    cluster/etc/cluster-env.csh
    cluster/etc/cluster-env.sh
    vnfs/libexec/wwmkchroot/functions

Signed-off-by: John L. Jolly <jjolly@suse.com>
Signed-off-by: Egbert Eich <eich@suse.de>
---
 cluster/etc/cluster-env.csh       | 2 --
 cluster/etc/cluster-env.sh        | 2 --
 vnfs/libexec/wwmkchroot/functions | 2 --
 3 files changed, 6 deletions(-)
diff --git a/cluster/etc/cluster-env.csh b/cluster/etc/cluster-env.csh
index 4631ad8..b5ef501 100755
--- a/cluster/etc/cluster-env.csh
+++ b/cluster/etc/cluster-env.csh
@@ -1,5 +1,3 @@
-#!/bin/tcsh
-
 if ( -x "/usr/bin/cluster-env" ) then
    /usr/bin/cluster-env
 endif
diff --git a/cluster/etc/cluster-env.sh b/cluster/etc/cluster-env.sh
index 74b6f87..5a5ea76 100755
--- a/cluster/etc/cluster-env.sh
+++ b/cluster/etc/cluster-env.sh
@@ -1,5 +1,3 @@
-#!/bin/sh
-
 if [ -x "/usr/bin/cluster-env" ]; then
    /usr/bin/cluster-env
 fi
diff --git a/vnfs/libexec/wwmkchroot/functions b/vnfs/libexec/wwmkchroot/functions
index 70e1ad8..fe3028f 100644
--- a/vnfs/libexec/wwmkchroot/functions
+++ b/vnfs/libexec/wwmkchroot/functions
@@ -1,5 +1,3 @@
-#!/bin/bash
-
 eval `wwconfig -as`
 
 MEDIA_MOUNTPATH="$CHROOTDIR/tmp/wwmkchroot.mount"
